tron/config/schema/tronfig_schema.json

```diff
@@ -0,0 +1,218 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "description": "",
+    "minProperties": 1,
+    "type": "object",
+    "additionalProperties": false,
+    "minProperties": 1,
```

Comment:
`min_properties: 2` because we always need `jobs` and `services`?

Reply:
I think we always need more than that actually. let me figure out which ones are actually required and how to write the json schema for that.